package/Shaders/Lighting.hlsl

```diff
@@ -1606,35 +1964,54 @@ PS_OUTPUT main(PS_INPUT input, bool frontFace
 
 		float intensityMultiplier = 1 - intensityFactor * intensityFactor;
 		float3 lightColor = PointLightColor[lightIndex].xyz * intensityMultiplier;
-
+		float lightShadow = 1.f;
 		if (PixelShaderDescriptor & _DefShadow) {
 			if (lightIndex < numShadowLights) {
-				lightColor *= shadowColor[ShadowLightMaskSelect[lightIndex]];
+				lightShadow *= shadowColor[ShadowLightMaskSelect[lightIndex]];
 			}
 		}
 
 		float3 normalizedLightDirection = normalize(lightDirection);
 
+		lightColor *= lightShadow;
+
+#			if defined(TRUE_PBR)
+		{
+			float3 pointDiffuseColor, coatPointDiffuseColor, pointTransmissionColor, pointSpecularColor;
+			float3 refractedLightDirection = normalizedLightDirection;
+			[branch] if ((PBRFlags & TruePBR_InterlayerParallax) != 0)
+			{
+				refractedLightDirection = -refract(-normalizedLightDirection, coatModelNormal, eta);
+			}
+			float3 pbrLightColor = AdjustDirectLightColorForPBR(lightColor);
```

Comment:
`AdjustDirectLightColorForPBR` is not defined

Reply:
Fixed